src/main/resources/config/liquibase/changelog/20190603142500_changelog.xml

```diff
@@ -1,33 +1,38 @@
-<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
-<changeSet id="201905026142600" author="behnke">
-    <!-- create new entity programming exercise test case -->
-    <createTable tableName="programming_exercise_test_case">
-        <column name="id" type="bigint" autoIncrement="${autoIncrement}">
-            <constraints primaryKey="true" nullable="false"/>
-        </column>
-        <column name="test_name" type="varchar(255)">
-            <constraints nullable="false" />
-        </column>
-        <column name="weight" type="integer">
-            <constraints nullable="false" />
-        </column>
-        <column name="active" type="boolean">
-            <constraints nullable="false" />
-        </column>
-        <column name="exercise_id" type="bigint">
-            <constraints nullable="true" />
-        </column>
-    </createTable>
-    <addUniqueConstraint tableName="programming_exercise_test_case" columnNames="test_name, exercise_id" constraintName="exercise_test_case" />
-    <!-- add foreign key constraint from test cases to programming exercises -->
-    <addForeignKeyConstraint baseColumnNames="exercise_id"
-                             baseTableName="programming_exercise_test_case"
-                             constraintName="fk_programming_exercise_test_case_exercise_id"
-                             referencedColumnNames="id"
-                             referencedTableName="exercise"/>
-    <!-- add sequential test run boolean flag to exercise -->
-    <addColumn tableName="exercise">
-        <column name="sequential_test_runs" type="Boolean"/>
-    </addColumn>
-</changeSet>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <property name="autoIncrement" value="true"/>
```

Comment:
This is the only line I changed, but I also ran an intellij autoformat (prettier does not support XML formatting).

Reply:
Can you revert the auto formatting please?

Reply:
I reverted the autoformat.